### PR TITLE
restore guillotina builds

### PR DESCRIPTION
**Pull request by jordic**

### CODE REVIEW

**Comment by jordic** [comment]
@aralroca @rboixaderg fixeu-vos que faig servir el packet aquest craco que em deixa incloure els sources del projecte en un altre projecte, i fa el build directament.

File: gmi/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"/home/jordi/projects/guillotina_react/gmi/src/index.js":"1","/home/jordi/projects/guillotina_react/gmi/src/App.js":"2","/home/jordi/projects/guillotina_react/gmi/src/reportWebVitals.js":"3"},{"size":478,"mtime":1610827812617,"results":"4","hashOfConfig":"5"},{"size":1476,"mtime":1610883428940,"results":"6","hashOfConfig":"5"},{"size":362,"mtime":1610826394285,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"3zwxsf",{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"/home/jordi/projects/guillotina_react/gmi/src/index.js",[],["15","16"],"/home/jordi/projects/guillotina_react/gmi/src/App.js",[],"/home/jordi/projects/guillotina_react/gmi/src/reportWebVitals.js",[],{"ruleId":"17","replacedBy":"18"},{"ruleId":"19","replacedBy":"20"},"no-native-reassign",["21"],"no-negated-in-lhs",["22"],"no-global-assign","no-unsafe-negation"]
```
Comment (aralroca) [inline gmi/.eslintcache:1]: should .eslintcache be in the gitignore?

File: gmi/src/reportWebVitals.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
```
Comment (aralroca) [inline gmi/src/reportWebVitals.js:3]: Interesting

**Review by aralroca — APPROVED** [review]

**Review by rboixaderg — APPROVED** [review]

**Comment by rboixaderg** [comment]
If gmi is another example app, shouldn't it be in the example folder?

**Comment by jordic** [comment]
it's not an example app, it's the default app shipped with guillotina :)

**Comment by jordic** [comment]
@rboixaderg @aralroca how I can apply the default formatter, there's any command I can run before doing a comit? Or some comit hook I can use?

**Comment by rboixaderg** [comment]
@jordic you can run 
```
yarn format 
```

but this command should execute before each commit.

```
"husky": {
    "hooks": {
      "pre-commit": "prettier --write \"src/**/*.js\""
    }
  },
```
We can use lint-staged to check only staged fields in pre-commit too

**Review by aralroca — APPROVED** [review]

**Comment by jordic** [comment]
will wait till @rboixaderg merges his PR, this way I can better focus on e2e cypress,

**Comment by rboixaderg** [comment]
Which PR @jordic ? This PR #96  it is ready to be merged, but I want to review how user can override EditComponent

**Comment by jordic** [comment]
Perhaps it's better to start with what you have and we will evolve it on the future. I'm still trying to figure out the best option to make guilotina overridable from outside... (We have a low level API, the registry, but I would love to have some kind of "external" via window.Registry, overrides... ), still not pretty sure.

**Comment by rboixaderg** [comment]
@aralroca @jordic I added search engine configuration and pass tests. If you want we cant publish a new version with jordi's changes and this last feature!

**Review by aralroca — APPROVED** [review]

**Pull request merged**